### DISCUSSION TITLE
我用 Docusaurus 建立了一個新的技術筆記網站！ | Bosh's Blog

### DISCUSSION THREAD

**Discussion by giscus** [Bosh-Kuo/Bosh-Hexo-Blog#3] [Announcements]
# 我用 Docusaurus 建立了一個新的技術筆記網站！ | Bosh's Blog

這篇文章宣布一個消息，我使用 Docusaurus 建立建立了一個新的技術部落格網站 Bosh’s Tech Notes，未來技術筆記類型的文章將會在新網站發布。其實很早以前我就開始觀望 Docusaurus 這個 SSG 了，但由於開始注意到 Docusaurus 時，已經使用 Hexo 幾個月，且當下使用 Hexo 也還算滿意的狀態。 由於目前對新站有比較明確的一些想法和目標，近期應該會比

https://blog.boshkuo.com/Docusaurus-Blog/

**Comment by Shuai-Xin** [comment]
_posts 資料夾裡面可以自己分類出多個資料夾才對，我個人是這樣分類的

**Reply by Bosh-Kuo** [reply]
我之前以為_posts只能放.md檔 謝謝你的回應 讓我知道一個有用的資訊~
